src/main/resources/com/google/api/codegen/nodejs/version_index.snip

```diff
@@ -49,10 +49,28 @@
     {@comments(@service.doc.lines)}
     *
     * @@param {object=} options - [Configuration object](#/docs).
+    * @@param {string=} options.key - API key to use form communication with the
+    *     service.
+    * @@param {string=} options.keyFilename - Path to a key file to use for an
+    *     API key.
+    * @@param {(string|number)=} options.logLevel - An integer from 0-5
+    *     inclusive or a decimal representation of an integer from 0-5 inclusive
+    *     in String form.
     * @@param {number=} options.port - The port on which to connect to
     *     the remote host.
+    * @@param {string=} options.projectId - The projectId of the project
+    *     deployed.
+    * @@param {object=} options.serviceContext - The service context of the
```

Comment:
I did a simple search through the toolkit and gax-nodejs for "serviceContext" and found nothing. Is this option api-specific?